integration-tests/deployment/ccip/view/v1_6/feequoter.go

```diff
@@ -138,6 +138,7 @@ func GetSupportedTokens(taContract *token_admin_registry.TokenAdminRegistry) ([]
 	return tokens, nil
 }
 
+// From the perspective of the OnRamp, the destination chains are the source chains for the OffRamp.
 func GetDestinationSelectors(routerContract *router1_2.Router) ([]uint64, error) {
```

Comment:
Thinking it might make more sense to have this defined in `v1_2/router.go`?

Reply:
actually maybe not since we're calling it `GetDestinationSelectors`

Reply:
How about `GetRemoteChainSelectors` ?